tests/Feature/UserLogin.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+
+trait userLogin
+{
+    protected function userLogin()
+    {
+        $user = User::factory()->create();
+        $this->actingAs($user);
+    }
+}
```

Comment:
you can make a common method on the test file if it's related to that particular test. and if it's used in all test files then move to pest.php file